packages/flutter_migrate/pubspec.yaml

```diff
@@ -0,0 +1,19 @@
+name: flutter_migrate
+description: A tool to migrate legacy flutter projects to modern versions.
+version: 0.0.1
+repository: https://github.com/flutter/packages/tree/main/packages/flutter_migrate
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3Ap%3A%20flutter_migrate
+publish_to: none
+
+environment:
+  sdk: '>=2.18.0 <3.0.0'
+
+dependencies:
+  args: 2.3.1
+  flutter_tools:
+    git:
+      url: https://github.com/flutter/flutter.git
+      ref: 58ec4839159ea24eafe200d9c1ff41e35f720fa0
+      path: packages/flutter_tools/
```

Comment:
Why do we need a git reference to flutter_tools? Can't we remove this for now and add it when you add Dart code that uses it so that the reason is clear?